Lib/statistics.py

```diff
@@ -910,10 +910,13 @@ def correlation(x, y, /):
         raise StatisticsError('at least one of the inputs is constant')
 
 
-def linear_regression(regressor, dependent_variable):
+LinearRegression = namedtuple('LinearRegression', ['intercept', 'slope'])
```

Comment:
The parameter order seems backwards to me, here and through out the PR.  Traditionally, we say an equation for a line in is slope-intercept form as `y = mx + b` with the *m* preceding the *b*.

Reply:
It depends on your background, [in statistics](https://en.wikipedia.org/wiki/Linear_regression#Introduction) the model is often written as y = β_0 + β_1 * x + ε , so the intercept is written before. Also, if now it is a named tuple, the order is of less importance. Of course, if you insist, I may change it.